### DISCUSSION TITLE
Add Related Collections to Many-to-Any Relationship

### DISCUSSION THREAD

**Discussion by notjonmadden** [directus/directus#4544] [Feature Requests]
Our app defines a set of Features that can be used to enable functionality in an app. The feature can be granted by a user's association with a number of different Collection types, so I add an M2A field, granted_by, to the Feature data model. When I later add a new Collection type, I want to extend granted_by to allow relation to the new type.

**Comment by rijkvanzanten** [comment]
I'm not sure I'm following what you mean by 

> When I later add a new Collection type, I want to extend granted_by to allow relation to the new type.

🤔

**Reply by notjonmadden** [reply]
Thanks for the response! I'm saying I'd like to add additional Related Collections here, *after* the field has been created.
![image](https://user-images.githubusercontent.com/3360335/111204989-1692a100-859d-11eb-8665-97c177f34c3a.png)

**Reply by rijkvanzanten** [reply]
That dropdown that says `2 Items` is active, and should be usable at this point 🙂

**Reply by notjonmadden** [reply]
Right, sorry I'm being unclear. The screenshot is from initial setup of the field. At that point, yes I can add as many Related Collections as I want. What I'm looking for is an ability to add more of those items later, after the field has already been created. Here's what I see when I try to do that:

![image](https://user-images.githubusercontent.com/3360335/111207350-e7316380-859f-11eb-956b-668cde94300d.png)

**Reply by rijkvanzanten** [reply]
Oh I see. Is your directus installation up to date? That was added in https://github.com/directus/directus/pull/4271 which made it into rc.45 🙂

**Reply by notjonmadden** [reply]
Ha, one step ahead of me 😅 I'm using cloud, which tells me I'm running rc.44

<img width="210" alt="image" src="https://user-images.githubusercontent.com/3360335/111208014-9e2ddf00-85a0-11eb-977d-ec827f8547d4.png">

**Reply by rijkvanzanten** [reply]
Ahh gotcha! That will be updated shortly 🙂 👍🏻

**Reply by benhaynes** [reply]
> Ha, one step ahead of me 😅 I'm using cloud, which tells me I'm running rc.44

Cloud has been updated to RC.50, you should be all set! We'll keep these updated moving forward, with only a small lag in versioning to ensure stability on the SaaS.
@WoLfulus has confirmed

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
